config/rhoai/ocp-4.17-addons/validating_admission_policy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: kueue-validating-admission-policy
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["ray.io"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["rayclusters"]
+  matchConditions:
+    - name: "skip-if-deleting"
+      expression: "!has(object.metadata.deletionTimestamp)"
+    - name: "skip-if-owned-by-appwrapper"
+      expression: "!has(object.metadata.ownerReferences) || object.metadata.ownerReferences[0].kind != 'AppWrapper'"
```

Comment:
I wonder if CEL supports wildcard to match array items?

Reply:
I found a special CEL function (`exists`) that does this. Thanks Antonin!